pkgs/applications/misc/mu-repo/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, buildPythonApplication, pytest, git }:
```

I prefer to leave out stden when possible.
```suggestion
{ lib, fetchFromGitHub, buildPythonApplication, pytest, git }:
```

---

Good point, fixed.